api/osrd_infra/models/train_schedule.py

```diff
@@ -2,16 +2,27 @@
 
 from osrd_infra.models import PathModel, RollingStock, Timetable
 from osrd_infra.schemas.rolling_stock import ComfortType
-from osrd_infra.schemas.train_schedule import MRSP, Allowances, TrainScheduleLabels, TrainScheduleOptions
+from osrd_infra.schemas.train_schedule import (
+    MRSP,
+    Allowances,
+    TrainScheduleLabels,
+    TrainScheduleOptions,
+)
 from osrd_infra.utils import PydanticValidator
 
 
-class TrainScheduleModel(models.Model):
+class TrainScheduleOutput(models.Model):
+    mrsp = models.JSONField(validators=[PydanticValidator(MRSP)])
+    base_simulation = models.JSONField()
+    eco_simulation = models.JSONField(null=True)
+    modes_and_profiles = models.JSONField(default=list)
+
+
+class TrainScheduleInput(models.Model):
```

Comment:
Why keep `TrainSchedule` in the name of both class ? Why not `TrainSchedule` and `SimulationResult` for example ?